crypto/fipsmodule/sha/keccak1600.c

```diff
@@ -102,422 +100,18 @@ static const uint64_t iotas[] = {
-    A[3][3] = ROL64(A[3][2], rhotates[3][2]);
-    A[4][4] = ROL64(A[4][1], rhotates[4][1]);
-
-    A[1][4] = ROL64(A[4][2], rhotates[4][2]);
-    A[2][3] = ROL64(A[3][4], rhotates[3][4]);
-    A[3][2] = ROL64(A[2][1], rhotates[2][1]);
-    A[4][1] = ROL64(A[1][3], rhotates[1][3]);
-
-    A[4][2] = ROL64(A[2][4], rhotates[2][4]);
-    A[3][4] = ROL64(A[4][3], rhotates[4][3]);
-    A[2][1] = ROL64(A[1][2], rhotates[1][2]);
-    A[1][3] = ROL64(A[3][1], rhotates[3][1]);
-
-    A[2][4] = ROL64(A[4][0], rhotates[4][0]);
-    A[4][3] = ROL64(A[3][0], rhotates[3][0]);
-    A[1][2] = ROL64(A[2][0], rhotates[2][0]);
-    A[3][1] = ROL64(A[1][0], rhotates[1][0]);
-
-    A[1][0] = ROL64(C[3],    rhotates[0][3]);
-    A[2][0] = ROL64(C[1],    rhotates[0][1]);
-    A[3][0] = ROL64(C[4],    rhotates[0][4]);
-    A[4][0] = ROL64(C[2],    rhotates[0][2]);
-
-    C[0] = A[0][0];
-    C[1] = A[1][0];
-    D[0] = A[0][1];
-    D[1] = A[1][1];
-
-    A[0][0] ^= (~A[0][1] & A[0][2]);
-    A[1][0] ^= (~A[1][1] & A[1][2]);
-    A[0][1] ^= (~A[0][2] & A[0][3]);
-    A[1][1] ^= (~A[1][2] & A[1][3]);
-    A[0][2] ^= (~A[0][3] & A[0][4]);
-    A[1][2] ^= (~A[1][3] & A[1][4]);
-    A[0][3] ^= (~A[0][4] & C[0]);
-    A[1][3] ^= (~A[1][4] & C[1]);
-    A[0][4] ^= (~C[0]    & D[0]);
-    A[1][4] ^= (~C[1]    & D[1]);
-
-    C[2] = A[2][0];
-    C[3] = A[3][0];
-    D[2] = A[2][1];
-    D[3] = A[3][1];
-
-    A[2][0] ^= (~A[2][1] & A[2][2]);
-    A[3][0] ^= (~A[3][1] & A[3][2]);
-    A[2][1] ^= (~A[2][2] & A[2][3]);
-    A[3][1] ^= (~A[3][2] & A[3][3]);
-    A[2][2] ^= (~A[2][3] & A[2][4]);
-    A[3][2] ^= (~A[3][3] & A[3][4]);
-    A[2][3] ^= (~A[2][4] & C[2]);
-    A[3][3] ^= (~A[3][4] & C[3]);
-    A[2][4] ^= (~C[2]    & D[2]);
-    A[3][4] ^= (~C[3]    & D[3]);
-
-    C[4] = A[4][0];
-    D[4] = A[4][1];
-
-    A[4][0] ^= (~A[4][1] & A[4][2]);
-    A[4][1] ^= (~A[4][2] & A[4][3]);
-    A[4][2] ^= (~A[4][3] & A[4][4]);
-    A[4][3] ^= (~A[4][4] & C[4]);
-    A[4][4] ^= (~C[4]    & D[4]);
-    A[0][0] ^= iotas[i];
-}
-
-static void KeccakF1600(uint64_t A[SHA3_ROW_SIZE][SHA3_ROW_SIZE])
-{
-    size_t i;
-
-    for (i = 0; i < 24; i++) {
-        Round(A, i);
-    }
-}
-
-#elif defined(KECCAK_2X)
-
- // This implementation is variant of KECCAK_1X above with outer-most
- // round loop unrolled twice. This allows to take temporary storage
- // out of round procedure and simplify references to it by alternating
- // it with actual data (see round loop below). Originally it was meant
- // rather as reference for an assembly implementation, but it seems to
- // play best with compilers [as well as provide best instruction per
- // processed byte ratio at minimal round unroll factor]...
-static void Round(uint64_t R[SHA3_ROW_SIZE][SHA3_ROW_SIZE], uint64_t A[SHA3_ROW_SIZE][SHA3_ROW_SIZE], size_t i)
+/*
+ * This implementation is variant of KECCAK_1X above with outer-most
+ * round loop unrolled twice. This allows to take temporary storage
```

Comment:
"above" doesn't apply anymore, I understand.
```suggestion
 * KECCAK_2X:
 * This is the default implementation used in OpenSSL and the fastest;
 * the other implementations were removed from this file.
 * This implementation is a variant of KECCAK_1X (see OpenSSL) with outer-most
 * round loop unrolled twice. This allows to take temporary storage
```